draft-ietf-httpbis-http2bis.xml

```diff
@@ -2021,10 +2022,10 @@
             <dt anchor="SETTINGS_MAX_HEADER_LIST_SIZE">SETTINGS_MAX_HEADER_LIST_SIZE (0x6):</dt>
             <dd>
               <t>
-                  This advisory setting informs a peer of the maximum size of header list that the
+                  This advisory setting informs a peer of the maximum size of header section that the
```

Comment:
Does this setting also apply to trailer sections?

Reply:
I thought we called this the "field section" above.